styles/global.css

```diff
@@ -3,3 +3,15 @@ body,
 #__next {
   height: 100%;
 }
+
+.nav-pills .active {
```

Comment:
Can't figure out an easy way to override bootstrap styles without using `!important` because of the way css import are getting ordered. Will see if I can figure out a way around this, but seems safe since it's only targeting nav elements and the custom css coverage is so low.